project_name/settings.py-tpl

```diff
@@ -75,7 +73,11 @@ WSGI_APPLICATION = '{{ project_name }}.wsgi.application'
 # https://docs.djangoproject.com/en/{{ docs_version }}/ref/settings/#databases
 
 DATABASES = {
-    "default": django_mongodb_backend.parse_uri("mongodb://localhost:27017/{{ project_name }}"),
+    'default': {
+        'ENGINE': 'django_mongodb_backend',
+        'HOST': 'mongodb://localhost:27017/',
```

Comment:
For this configuration, "HOST" isn't needed. `'mongodb://localhost:27017/'` basically only serves as quasi-documentation to show that you can put a connection string here, but may give some wrong impressions, like the fact that a port is required, so maybe we should rethink this.